narwhal/worker/src/batch_maker.rs

```diff
@@ -34,43 +46,47 @@ pub struct BatchMaker {
     /// Receive reconfiguration updates.
     rx_reconfigure: watch::Receiver<ReconfigureNotification>,
     /// Channel to receive transactions from the network.
-    rx_batch_maker: Receiver<Transaction>,
+    rx_transaction: Receiver<(Transaction, TxResponse)>,
     /// Output channel to deliver sealed batches to the `QuorumWaiter`.
-    tx_quorum_waiter: Sender<Batch>,
-    /// Holds the current batch.
-    current_batch: Batch,
-    /// Holds the size of the current batch (in bytes).
-    current_batch_size: usize,
+    tx_message: Sender<(Batch, Option<tokio::sync::oneshot::Sender<()>>)>,
```

Comment:
Similarly, `tx_message` was renamed to `tx_quorum_waiter` for consistency. I fix this later.

Reply:
Renamed to `rx_batch_maker`!